### ISSUE TITLE
extra characters returned at end of SPF lookup

### ISSUE THREAD

**Issue by simsong** [getdnsapi/getdns-python-bindings#32]
This is using the `examples/get-spf.py` program provided in the getdns distribution:

```
[ec2-user@ec2 examples]$ python27 get-spf.py yahoo.com
TTL:1800   NAME:yahoo.com.    TXT:['v=spf1 redirect=_spf.mail.yahoo.comm']
```

Notice the extra `m` at the end of the TXT record.

It is not there if you use `dig`:

```
[ec2-user@ec2 examples]$ dig -t txt yahoo.com

; <<>> DiG 9.8.2rc1-RedHat-9.8.2-0.30.rc1.38.amzn1 <<>> -t txt yahoo.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 54376
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 0

;; QUESTION SECTION:
;yahoo.com.			IN	TXT

;; ANSWER SECTION:
yahoo.com.		1800	IN	TXT	"v=spf1 redirect=_spf.mail.yahoo.com"

;; Query time: 5 msec
;; SERVER: 172.30.0.2#53(172.30.0.2)
;; WHEN: Mon Aug 17 14:09:17 2015
;; MSG SIZE  rcvd: 75

[ec2-user@ec2 examples]$ 
```

**Comment by MelindaShore** [comment]
I've just pushed a fix to the develop branch.  I'm planning on a major bindings release next week that will include this fix, the zombie fix, Python 3 support, callables as callbacks, and some other bugfixes and features.

**Comment by simsong** [comment]
Great. Thanks!

**Comment by MelindaShore** [comment]
Fix released with v0.4.0
